src/commands/AudacityCommand.cpp

```diff
@@ -63,7 +63,7 @@ AudacityCommand::~AudacityCommand()
 }
 
 
-PluginPath AudacityCommand::GetPath(){        return BUILTIN_GENERIC_COMMAND_PREFIX + GetSymbol().Internal(); }
+PluginPath AudacityCommand::GetPath() const {        return BUILTIN_GENERIC_COMMAND_PREFIX + GetSymbol().Internal(); }
```

Comment:
I notice now that the declarations in this class are missing the override specifier.  It would be nice to add one commit that supplies that, but not essential.